### ISSUE TITLE
Add "drip avoider" drawing option.

### ISSUE THREAD

**Issue by techninja** [evil-mad/robopaint#210]
Would send the paintbrush down the line to the bottom of the tray to hit the bottom and (hopefully) shaking any drops loose.

* Would this work? Dunno.
* Will it keep drops from getting in the middle of the paper? Maybe.
* Is it worth the effort? Maybe not.

**Comment by oskay** [comment]
I have not had a problem with drips. Perhaps a gentler inking cycle is in order instead?

**Comment by techninja** [comment]
Agreed. Though if there's too much water in a paint pot, apart from using a napkin, it's tricky to manage.
